alexandria/core/views.py

```diff
@@ -100,6 +108,44 @@ def update(self, request, *args, **kwargs):
 
         return response
 
+    @action(methods=["get"], detail=True)
+    def convert(self, request, pk=None):
+        if not settings.ALEXANDRIA_ENABLE_PDF_CONVERSION:
+            raise ValidationError(_("PDF conversion is not enabled."))
+
+        document = self.get_object()
+        file = document.get_latest_original()
+
+        response = requests.post(
+            settings.ALEXANDRIA_DMS_URL + "/convert",
+            data={"target_format": "pdf"},
+            headers={"authorization": get_authorization_header(request)},
+            files={"file": file.content},
+        )
+
+        if response.status_code == HTTP_401_UNAUTHORIZED:
+            raise AuthenticationFailed(_("Token has expired."))
```

Comment:
Let's return the error message that the DMS responds with.

Reply:
Done

Reply:
DMS will return JSON, no?

Reply:
Parsing the response now